src/components/header/export-modal/renderer.tsx

```diff
@@ -53,15 +55,18 @@ class ExportModal extends React.PureComponent<Props, State> {
       method: 'post',
       mode: 'cors',
     });
+    if (pdf.status !== 200) {
+      this.setState({loadingPDF: false, errorLoadingPdf: true});
```

Comment:
Instead of just a boolean, do you think it would be better to get an actual error message and display that?

Reply:
There may be errors other than external URLs. Please revise in a follow up pull request.